packages/web/src/components/gcds-search/gcds-search.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, Host, Prop, Element, State, Event, EventEmitter, h } from '@stencil/core';
+import { assignLanguage, observerConfig } from '../../utils/utils';
+import I18N from './i18n/I18N'
+
+@Component({
+  tag: 'gcds-search',
+  styleUrl: 'gcds-search.css',
+  shadow: false,
+  scoped: true,
+})
+export class GcdsSearch {
+  @Element() el: HTMLElement;
+
+  /**
+  * Set the placeholder and label for the search input. Becomes "Search [placeholder]"
+  */
+  @Prop() placeholder: string = "Canada.ca";
+
+  /**
+  * Sets the action for the search form. Default will be canada.ca global search
+  */
+  @Prop() action: string = "/sr/srb.html";
+
+  /**
+  * Set the form method of the search form
+  */
+  @Prop() method: "get" | "post" = "get";
+
+  /**
+  * Set the name of the search input
+  */
+  @Prop() name: string = "q";
+
+  /**
+  * Set the name of the search input
+  */
+  @Prop() searchId: string = "search";
+
+  /**
+  * Set a list of predefined search terms
+  */
+  @Prop() suggested: Array<string>;
+
```

Comment:
Can you add a comment here marking the start of the events? Like here:
https://github.com/cds-snc/gcds-components/blob/main/packages/web/src/components/gcds-button/gcds-button.tsx#L143